cdap-validation/bin/ambari.py

```diff
@@ -0,0 +1,87 @@
+import urllib2
+import json
+import re
+import glob, os
+import test_helpers as helpers
+
+# Start Ambari configuration retrieval process
```

Comment:
Give methods a proper header, including:
- Description
- All arguments and their expected types and input
- Any return state (value, object, etc)